### PR TITLE
Example API calls for finnhub.io, improve azure-pipelines

### PR DESCRIPTION
**Pull request by Necktschnagge**

### CODE REVIEW

**Comment by Necktschnagge** [comment]
I've just made some additional requests which tidy up code a little bit.

**Comment by Necktschnagge** [comment]
Before [fb80114](https://github.com/Necktschnagge/finn/pull/11/commits/fb8011454bd46e2e2f1c43e3f9cfd9d0267b1fab) the application indeed crashed if there was no `summary.json` already created before app start.

**Comment by Necktschnagge** [comment]
TODOs:
* add some other relevant API endpoints to `finnhub_rest_client`

**Comment by Necktschnagge** [comment]
All requests are nearly copy-paste. Should introduce some fascade pattern...
Also every request should run through an api limit exceed checker like stated here: #15

**Review by r0ost3r — DISMISSED** [review]

**Comment by Necktschnagge** [comment]
I just renamed the get requests of `finnhub_rest_client` so that the are equal to the section headers of the API documentation https://finnhub.io/docs/api/company-profile2

File: src/utility.h [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	return default_json;
+}
+
+inline void save_json(const std::string& file_name, const nlohmann::json& json, int indent = 1) {
+	std::ofstream file;
+	file.open(file_name);
+	file << std::setw(indent) << json << std::endl;
+}
+
+inline void try_sort_stock_list(nlohmann::json& stock_list) {
```
Comment (Necktschnagge) [inline src/utility.h:42]: make this exception-safe!

**Comment by Necktschnagge** [comment]
The runtime check is weird. We should introduce a API retry limit given to finn client on construction.
The server routine then can finish within a few minutes.

**Comment by Necktschnagge** [comment]
Minor, pedantic improvement requests will be moved into follow-up issues!

**Pull request merged**